cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -135,8 +135,10 @@ def test_total_services(appliance, setup_provider, context, order_service):
     """Tests total services count displayed on dashboard.
 
     Polarion:
-        assignee: sshveta
+        assignee: nansari
+        casecomponent: SelfServiceUI
         initialEstimate: 1/4h
+        tags: service
```

Comment:
Q: why do you have different tags for ssui tests? Here it's `service`, later it's `ssui`

Reply:
thanks for pointing.making changes